utils/cpplint_files.txt

```diff
@@ -60,6 +60,7 @@ src/artm/core/data_loader.h
 src/artm/core/dense_phi_matrix.h
 src/artm/core/dictionary.h
 src/artm/core/exceptions.h
+src/artm/core/fileread_helpers.h
```

Comment:
Could you move this file under `src/utility` and rename to `ifstream_or_cin.h`? This would discourage people from putting more stuff into this file, and generally this is a good practice...